shared/actions/gregor.js

```diff
@@ -174,13 +172,10 @@ function * handleKbfsFavoritesOOBM (kbfsFavoriteMessages: Array<OutOfBandMessage
   }).filter(i => !!i)
 }
 
-function * handlePushOOBM (pushOOBM: pushOOBM) {
+function * _handlePushOOBM (pushOOBM: pushOOBM) {
```

Comment:
i guess we do this inconsistently. i think we're mostly on board to do this for method in classes to mark them as not an external interface. We also do this sometimes just in a file to mark whats exposed or not. i can remove it